cmd/podman/containers/cp.go

```diff
@@ -115,6 +117,110 @@ func doCopy(funcA func() error, funcB func() error) error {
 	return errorhandling.JoinErrors(copyErrors)
 }
 
+func copyContainerToContainer(sourceContainer string, sourcePath string, destContainer string, destPath string) error {
+	if err := containerMustExist(sourceContainer); err != nil {
+		return err
+	}
+
+	if err := containerMustExist(destContainer); err != nil {
+		return err
+	}
+
+	sourceContainerInfo, err := registry.ContainerEngine().ContainerStat(registry.GetContext(), sourceContainer, sourcePath)
+	if err != nil {
+		return errors.Wrapf(err, "%q could not be found on container %s", sourcePath, sourceContainer)
+	}
+
+	var destContainerBaseName string
+	destContainerInfo, destContainerInfoErr := registry.ContainerEngine().ContainerStat(registry.GetContext(), destContainer, destPath)
```

Comment:
NIT: Why not just use `err` here?

Reply:
I see below you used the err, but it is very hard to follow.  Could you create a boolean variable with a name indicating what you intend.

Reply:
Can I tackle this and the other comment in another PR?  Now it's consistent with the other two copy functions but I can spin up a follow-up PR trying to simplify things a bit more.